src/components/Hero.js

```diff
@@ -48,34 +48,16 @@ function Hero({handleLogIn}) {
               <small>Trusted By</small>
             </li>
             <li>
-              <img
-                alt="graphql"
-                src={graphql}
-              />
+              <img alt="graphql" src={graphql} />
```

Comment:
@bdougie This is I'm talking about, Link to GraphQL website!

Reply:
![Screenshot](https://user-images.githubusercontent.com/61746440/83329879-7f788900-a2a9-11ea-99a2-86033dd4e4db.png)

@bdougie for this!